opal/mca/accelerator/cuda/accelerator_cuda.c

```diff
@@ -479,6 +512,11 @@ static int accelerator_cuda_get_address_range(int dev_id, const void *ptr, void
 static int accelerator_cuda_host_register(int dev_id, void *ptr, size_t size)
 {
     CUresult result;
+    result = opal_accelerator_cuda_delayed_init();
```

Comment:
result is a CUresult but is assigned an OPAL error code. Below, a CUDA error code is returned as OPAL error code (https://github.com/open-mpi/ompi/pull/11253/files#diff-39b9e960ab651004b6fb3f54ce4fa24ec51d4c91d9028e8a8d5bbd69ba127830R528)